framework/src/utils/MooseUtils.C

```diff
@@ -31,10 +31,42 @@
 
 // System includes
 #include <sys/stat.h>
+#include <numeric>
 
 namespace MooseUtils
 {
 
+int
```

Comment:
Levenshtein distance is always positive. Why `int`?

Reply:
I pushed a commit with the source where I copied the code from.  Since I didn't write the code, I'd err on the side of not changing it, but I don't feel strongly either way.  int makes it easier to test that this function never returns negative values though :-P

Reply:
To test that it _never_ returns negative values, you'd have to have it calculate _every_ possible distance. Good luck with that...